Autopilot/Utilities.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Data.SqlClient;
+using System.Windows;
+
+namespace Autopilot
+{
+    class Utilities
+    {
+        public string DBconnStrg = Properties.Settings.Default.AutopilotConnectionString; //zentraler DB-ConnectionStrg
+
+        public void SQLCmd(string SQLCmd_txt)
+        {
+            SqlConnection conn = new SqlConnection(DBconnStrg);
+            conn.Open();
+            SqlCommand cmd = new SqlCommand();
+            cmd.Connection = conn;
+            cmd.CommandText = SQLCmd_txt;
+            cmd.CommandType = System.Data.CommandType.Text;
+
+            try
+            {
+                cmd.ExecuteNonQuery();
+            }
+            catch (System.Exception err)
+            {
+                MessageBox.Show("Fehlermeldung: " + err.Message, "Fehler", MessageBoxButton.OK, MessageBoxImage.Error);
```

Comment:
Fehlerdarstellung bau ich noch in die GUI ein, das sollte nicht in ner MessageBox passieren ;)